_posts/2022-06-20-notes-from-community-call.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Notes from the Foundation for Public Code community call - 16 June 2022"
+date: 2022-06-08
```

Comment:
This date should be the same as the date in the file name, the publication date. I think the blog might be confused if they differ.